### ISSUE TITLE
Color based metal tank keys

### ISSUE THREAD

**Issue by liach** [CovertJaguar/Railcraft-Localization#346]
They are missing localization even in en_us now. Need to fix this at some point, preferably before release.

**Comment by CovertJaguar** [comment]
Do we remove the color labels entirely?

**Comment by liach** [comment]
we did not have color in the localization tags for metal tanks before.

**Comment by CovertJaguar** [comment]
Indeed. And I'm fine with it staying that way. Which means the Item class probably needs adjusting.
